### DISCUSSION TITLE
Is the OMS community version code open source

### DISCUSSION THREAD

**Discussion by laglangyue** [oceanbase/oceanbase#1297] [Q&A] [answered]
Is the OMS community version code open source

**Comment by whhe** [comment, marked as answer]
The source code of OMS CE is not open for now, but we have a plan to share a part of code of the suite in this year.

**Reply by laglangyue** [reply]
I am a data integration developer and look forward to the open source of OMS

**Comment by leilinen** [comment]
Hi, OMS CE now was opened source ? I am searching too.

**Reply by whhe** [reply]
No, it's not open source yet.
